.gitattributes

```diff
@@ -1,2 +1,3 @@
 *.java text=auto
 *.xml text=auto
+* text eol=lf
```

Comment:
Please revert this.

Reply:
It looks like there is still a difference, perhaps a newline was removed?

Reply:
oh yes you are right, sorry for that